sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java

```diff
@@ -76,13 +72,9 @@
      * @throws SQLException failed to execute SQL, throw this exception
      */
     public static void executeUpdate(final Connection connection, final String sql, final Collection<Object> params) throws SQLException {
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
```

Comment:
Not need comma for last line of try with resource statement

Reply:
ok